crates/sozo/src/ops/migration/mod.rs

```diff
@@ -112,7 +113,10 @@ async fn setup_env(
 
         let address = account.address();
 
-        config.ui().print(format!("\nMigration account: {address:#x}\n"));
+        config.ui().print(format!("\nMigration account: {address:#x}"));
+        if let Some(name) = name {
+            config.ui().print(format!("\nWorld seed: {name}\n"));
```

Comment:
Let's use "World name " here to be consistent with the cli arg name @lambda-0x

Reply:
https://github.com/dojoengine/dojo/pull/801